container.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Dashboard</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
+    
+    <link rel="stylesheet" href="container.css">
+</head>
+<body>
+  <div class="container"> 
+    <!-----Navbar-------> 
+    <nav class="navbar">
+        <div class="left-navbar-items">
+            <a href="#" class="navbar-brand">
+               <img class="logo" src="assets/images/hr-logo.png" alt="Logo" >
+               <span id="magazine">Magazine</span>
+            </a>
+            <p id="user-type">Admin</p>
+        </div>
+        <div class="right-navbar-icons">
+            <button type="button" class="icon-btn" title="Toggle Theme" id="theme-toggle-btn">
+                <i class="fas fa-sun"></i> 
+            </button>
+            <img class="avatar" src="assets/images/avatar-default.png" alt="Avatar" >

```

Comment:
also i did not see any avatar image, one little hint you can search for svg images to add rather than pngs or jpeg. they are better and have very high resolution, also you can use icons from either font awesome or bootstrap icons